websites/L/Learnjavascript/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "293803241110503426",
+		"name": "weemizo#7940"
+	},
+	"service": "Learnjavascript",
+	"description": {
+		"en": "The easiest way to  learn  &  practice  modern JavaScript. Learn in an interactive environment. Read short lessons, take notes, and complete challenges directly in your browser.",
+		"pl": "Najłatwiejszy sposób nauki i ćwiczenia nowoczesnego Javascript'u. Nauka w interaktywnym środowisku. Czytaj krótkie lekcje, pisz notatki i ukończ zadania bezpośrednio w twojej przeglądarce."
+	},
+	"url": "learnjavascript.online",
+	"version": "1.0.1",
+	"logo": "https://i.imgur.com/ykPXse3.png",
```

Comment:
Logos should be 512x512 at minimum. Either find a larger icon or use a program like waifu2x to enlarge it.